dockstore-integration-testing/src/test/java/io/dockstore/webservice/TokenResourceIT.java

```diff
@@ -79,6 +80,10 @@
 
     private TokenDAO tokenDAO;
     private UserDAO userDAO;
+    private static long initialTokenCount;
```

Comment:
This is confusing as a static variable and can lead to dependencies between tests. 
Turn this into a local variable (i.e. check token account at the beginning of tests)